packages/web-components/src/components/ic-typography/ic-typography.stories.mdx

```diff
@@ -23,7 +23,7 @@ import readme from "./readme.md";
 ### Variants
 
 <Canvas>
-  <Story name="Variants">
+  <Story name="Variants" parameters={{ loki: { skip: true } }}>
```

Comment:
The loki skip on this and vertical margins can be removed now that a fix was merged for the visual tests failing 😊